hangman.rb

```diff
@@ -0,0 +1,49 @@
+
+$random_words = ["ruby", "hangman", "flutter", "developer", "omkar", "github", "josh", "assignment"]
+
+def choose_word
+  $random_words.sample
+end
+
+def display_word(word,correct_guesses)
+  word.chars.map { |char| correct_guesses.include?(char) ? char : '_' }.join(' ')
+end
+
+def won?(word,correct_guesses)
+  word.chars.all? { |char| correct_guesses.include?(char) }
```

Comment:
Can this be optimized further?
We are iterating over all characters each time, check if we can skip it.